front/src/views/Artwork_list.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div class="about">
+    <div class="container">
```

Comment:
The mockup as a title : 
![image](https://github.com/Museum-GO/MuseumGO-App/assets/59999324/4c7025b3-84bb-4762-bad5-2dc10b641c6f)
Fix now or create an issue for later.